hudi-utilities/src/main/java/org/apache/hudi/utilities/UtilHelpers.java

```diff
@@ -191,15 +190,11 @@ public static SchemaPostProcessor createSchemaPostProcessor(
 
   }
 
-  public static Option<Transformer> createTransformer(List<String> classNames) throws IOException {
+  public static Option<Transformer> createTransformer(Option<List<String>> classNamesOpt) throws IOException {
     try {
-      List<Transformer> transformers = new ArrayList<>();
-      for (String className : Option.ofNullable(classNames).orElse(Collections.emptyList())) {
-        transformers.add(ReflectionUtils.loadClass(className));
-      }
-      return transformers.isEmpty() ? Option.empty() : Option.of(new ChainedTransformer(transformers));
+      return classNamesOpt.map(classNames -> classNames.isEmpty() ? null : new ChainedTransformer(classNames));
```

Comment:
null -> Option.empty().

Reply:
We can only return null inside `map` function as it expects the value. I can add a new API with something like `mapIf` if we want to clean this code further.